docs/_writing-tests/file-names.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: File Name Flags
+order: 2
+---
+
+The test filename is significant in determining the type of test it
+contains, and enabling specific optional features. This page documents
+the various flags avaiable and their meaning.
+
+### Test Type
+
+These flags must be the last element in the filename before the
+extension e.g. `foo-manual.html` will indicate a manual test, but
+`foo-manual-other.html` will not.
+
+<dl>
+  <dt>`-manual`
+  <dd><p>Indicates that a test is a non-automated test.
+  <dt>`-support`
+  <dd><p>Indicates that a file is not a test but a support file.
```

Comment:
Should we note that this is not needed if the resource is in a resources/ directory?